src/concrete/ml/onnx/convert.py

```diff
@@ -158,15 +158,17 @@ def get_equivalent_numpy_forward_from_torch(
 
 def get_equivalent_numpy_forward_from_onnx(
     onnx_model: onnx.ModelProto,
-    check_model: bool = True,
```

Comment:
this is a breaking change, please keep that parameter, I initially suggested to keep this and add a fixme below, we cannot remove parameters from public functions like this !

Reply:
I asked if I can remove it or not: https://zama-ai.slack.com/archives/C0455QDGT6C/p1701784027573399
Since we cannot remove public parameters like this.
I will add a fixme.